content/en/blog/_posts/2020-12-16-third-party-device-metrics-hits-ga.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: blog
+title: 'Third Party Device Metrics Hits G.A'
+date: 2020-12-16
+slug: third-party-device-metrics-hits-ga
+---
+
+ **Authors:** Renaud Gaubert, David Ashpole, and Pramod Ramarao
```

Comment:
Typically we add company attributions, eg:
```markdown
**Authors:** Renaud Gaubert (ACME); David Ashpole (FooBar Inc); Pramod Ramarao (BarFoo Systems).
```

Reply:
Renaud Gaubert (NVIDIA); David Ashpole (Google); Pramod Ramarao (NVIDIA)

Reply:
Added